target/linux/ramips/dts/WIDORA.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "mediatek,widora", "mediatek,mt7628an-soc";
+	model = "Widora NEO";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "mediatek:orange:wifi";
```

Comment:
Please, follow general convention and use labels with board name, like in many others DTS files.